operator/templates/node-scripts.yaml

```diff
@@ -33,3 +33,48 @@ data:
     dc=$CASSANDRA_DATACENTER
     rack=$RACK
     EOF
+  node-token-save.sh: |
+    # Used to capture the token map from a newly created cluster and save it
+    # Not used at the moment
+    if [ ! -f /var/lib/cassandra/token_map ]; then
+        while [ -z $NODE_TOKENS ]; do
+            NODE_ID=`nodetool info | grep ID | sed -n -e 's/ID[[:space:]]*\:[[:space:]]*\(.*\)$/\1/p'`
+            NODE_IP=`nodetool status | grep $NODE_ID | sed -n -e 's/UN[[:space:]]*\([0-9.]*\)[[:space:]]*.*/\1/p'`
```

Comment:
Nit: Do we have to run `nodetool info` and `nodetool status` for every iteration of the while-loop? How about running them once and only run `nodetool ring` in the loop? I'm not sure how many iterations such a loop has, but `nodetool` is quite expensive and this could reduce performance if there are many iterations.

Reply:
tbh, I'm not sure anymore. I think I've used that in one iteration of development to save the token_map. The while loop is there so the script waits until the cluster is fully up and running, but... Yeah, that could probably be optimized. 

The script is currently not used, i just didn't want to delete it yet, as figuring out the sed regexps for the fields was a PITA. 
I'll probably delete the script once the branch is merged - then it's still available in the git history.